.github/workflows/pr-python.yml

```diff
@@ -9,14 +9,16 @@ on:
 
 env:
   AWS_REGION: us-east-1
+  #copied the core repo sha from here:https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.github/workflows/test.yml#L9
+  CORE_REPO_SHA: ece57231cde5bb5656775a62f027cab0b1c76fce
 
 jobs:
   PR-Python:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        python: [3.8, 3.9]
+        python: ['3.8', '3.9']
```

Comment:
Why do we need to do this?

Reply:
The usage of quotes is recommended by [setup-python](https://github.com/actions/setup-python#basic-usage) in gh actions. It would cause error when suppose a version `3.10` has to be used.